app/models/channel.rb

```diff
@@ -3,9 +3,17 @@ class Channel < ApplicationRecord
   has_and_belongs_to_many :device_groups
   has_and_belongs_to_many :contents
 
+  after_destroy :send_email_notification
+
   validates :name, presence: true,
                    length: { minimum: 2,
                              maximum: 255 }
 
   scope :by_name, -> { order(:name) }
+
+  private
+
+  def send_email_notification
```

Comment:
unclear method name, `notify_deleted` or something like that would give more meaning for readers

Reply:
yes, rename. As a developer I MUST understand what method is doing by it's name